libretro-common/include/libretro.h

```diff
@@ -976,6 +977,85 @@ struct retro_hw_render_context_negotiation_interface
                                             * so it will be used after SET_HW_RENDER, but before the context_reset callback.
                                             */
 
+/* Seek flags from stdio.h */
+enum retro_seek_flag
+{
+  RETRO_SEEK_SET = 0, /* Beginning of file */
+  RETRO_SEEK_CUR = 1, /* Current position of the file pointer */
+  RETRO_SEEK_END = 2, /* End of file - implementation not required */
```

Comment:
If implementation isn't required then we shouldn't have it at all.

I vote remove this entire enum and seek only from start; the only usecase I can see for any non-SET is to append to a file, and that's so rare we don't need add more complexity. Just use vfs->get_file_length.